aspnetcore/security/key-vault-configuration/samples_snapshot/Startup.cs

```diff
@@ -0,0 +1,27 @@
+public class PrefixKeyVaultSecretManager : IKeyVaultSecretManager
+{
+    private readonly string _prefix;
+
+    public PrefixKeyVaultSecretManager(string prefix)
+    {
+        _prefix = $"{prefix}-";
+    }
+
+    public bool Load(SecretItem secret)
+    {
+        // Load a vault secret when its secret name starts with the 
+        // prefix. Other secrets won't be loaded.
+        return secret.Identifier.Name.StartsWith(_prefix);
+    }
+
+    public string GetKey(SecretBundle secret)
+    {
+        // Remove the prefix from the secret name and replace two 
```

Comment:
Comments are OK if they're repeated in the doc. They aren't localized. I tend to remove them. Often I have embedded snippets so after showing all the code I can show the commented code in a sub-snippet with comments in the text so it's localized.

Reply:
That's leftover from ancient times when we were more flexible. I, too, almost always remove them.

I'll check this in the morning and see if they need work.